tc/sgx/trusted_worker_manager/enclave_untrusted/enclave_bridge_wrapper/base_enclave_info.cpp

```diff
@@ -66,3 +66,10 @@ BaseEnclaveInfo::~BaseEnclaveInfo() {
     } catch (...) {}
 
 }  // BaseEnclaveInfo::~BaseEnclaveInfo
+
+// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+void BaseEnclaveInfo::terminate_enclave() {
+    try {
+        this->~BaseEnclaveInfo();
```

Comment:
Good practice is to not to call the constructor explicitly, it will called automatically when object goes out of scope or when we call through delete operator. if this object is used a local variable then there chances to do double deletion. Please check this. https://isocpp.org/wiki/faq/dtors#dont-call-dtor-on-local

Reply:
Introducing this call to terminate the enclave out of order.

Reply:
Removed the commit altogether as of now. Will introduce once behavior is verified for KME replication.